examples/python/invoke-rest-api/README.MD

```diff
@@ -0,0 +1,135 @@
+## About
+
+This function aims to provide a easy way to make a HTTP Post request to an API Endpoint that might help with a lot of integration scenarios. Eg. Post to a Slack channel, Create a PagerDuty or ServiceNow incident
+
+### Get the example function
+
+Clone this repository which contains the example functions. 
+
+```bash
+git clone https://github.com/vmware-samples/vcenter-event-broker-appliance
+cd vcenter-event-broker-appliance/examples/python/invoke-rest-api
+git checkout master
+```
+
+### Customize the function
+
+There are three key files, that you might have to modify if you are looking to customize this function and make a post api call to an external system. 
```

Comment:
Can you please also describe the case for modifying the other two files (stack/handler) or when that would be required? So it's easier to follow for newcomers.

Reply:
will do, this will be a long readme :)